src/Sets/SparsePolynomialZonotope.jl

```diff
@@ -207,6 +207,43 @@ indexvector(P::SPZ) = P.idx
 Returns a collection of n unique identifiers (intergers 1, …, n).
 """
 uniqueID(n::Int) = 1:n
+
+"""
+    mergeID(P1::SparsePolynomialZonotope, P2::SparsePolynomialZonotope)
+
+Returns two polynomial zonotopes equivalent to the correspondent input but with the same
+index vector.
+
+### Input
+
+- `P1` -- sparse polynomial zonotope
+- `P2` -- sprase polynomial zonotope
+
+### Output
+
+two polynomial zonotopes with the same index vector
+"""
+function mergeID(P1::SparsePolynomialZonotope, P2::SparsePolynomialZonotope)
+
+    idx1 = indexvector(P1)
+    idx2 = indexvector(P2)
+    idx1 == idx2 && return (P1, P2)
+
+    H = setdiff(idx2, idx1)
+    idx_new = vcat(idx1, H)
+    @show idx_new
```

Comment:
Remove this.

Reply:
now you know how I actually debug code :) 😅

Reply:
You should give `Infiltrator.jl` a try. There's no way back!

Reply:
This line is still there.

Reply:
strange, I thought I had removed, now should be gone